jenkins-pipelines/longevity-streams-non-topology-changing-nemesis.jenkinsfile

```diff
@@ -0,0 +1,11 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-alternator-streams-100gb-12h.yaml", "configurations/alternator/streams-with-nemesis.yaml", "configurations/alternator/streams-with-non-topology-changing-nemesis.yaml"]''',
```

Comment:
Better not to do it like this and have only one configuration.
It would be really hard to track changes later.